gmusicapi/clients/musicmanager.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import print_function, division, absolute_import, unicode_literals
-from future import standard_library
-standard_library.install_aliases()
+from future.utils import PY3
 from past.builtins import basestring
 from builtins import *  # noqa
+
 import os
 from socket import gethostname
 import time
-import urllib.request
-import urllib.parse
-import urllib.error
 from uuid import getnode as getmac
 import webbrowser
 
+if PY3:
```

Comment:
I guess, what's the point in reinventing the PY2 vs PY3 imports if there's already a compat layer in the requirements? I can live with either one, especially since I'm more a consumer of this library than a contributor... I'd just (personally) lean towards using a compat layer that already exists, rather than essentially rolling our own.